### PR TITLE
camera-imu-calibration readme file

### PR DESCRIPTION
**Pull request by shubhamgarg1994**
This tutorial briefly explains the kalibr tool installation.

### CODE REVIEW

File: wiki/sensing/camera-imu-calibration.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+```
+./kalibr_calibrate_cameras --models pinhole-equi pinhole-equi --topics /zed/left/image_raw /zed/right/image_raw --bag camera-imu-data.bag --target aprilgrid_6x6.yaml
+```
+Arguments:<br/>
+-models: Most typically used stereo cameras are of pinhole camera model. For more detail on the supported models, refer to this [link](https://github.com/ethz-asl/kalibr/wiki/supported-models).<br/>
+-topics: Name of the recodered camera topics  <br/>
+-bag: ROS bag containing the image and IMU data <br/>
+-target: yaml configuration file for the used target <br/>
+
+This command will generate a new yaml file with camera intrinsic calibration parameters. Above command can be skipped if the camera calibration yaml is already available. Now run below imu-camera calibration script.
```
Comment (smbryan) [inline wiki/sensing/camera-imu-calibration.md]: HTML tags should be avoided and aren't necessary here. 

However, I think you'd prefer adding a space after each `-`. For instance, `- models` not `-models` to create bullet points.

If you're having trouble with lines merging, then use two empty lines to distinguish separate paragraphs.

**Comment by smbryan** [comment]
@shubhamgarg1994, well written tutorial. There are some formatting points to tighten up before I can accept this.

**Comment by shubhamgarg1994** [comment]
Thanks @smbryan  for the comments. I have corrected the formatting error. Please let me know if it's correct now.

**Comment by smbryan** [comment]
@shubhamgarg1994 this looks good on my end. Give it 30 mins, then go checkout the updates on the site.

Good luck with finals!

**Pull request merged**